.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish package
+
+on:
+  push:
+    tags:
+      - '**'
+
+jobs:
+  publish-package:
+    name: Publish package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set env
+        run: echo "LIMINAL_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.6'
+      - name: Install dependencies for setup
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine
+      - name: Build and publish
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+          TWINE_REPOSITORY_URL: https://upload.pypi.org/legacy/
+        run: |
+          python setup.py sdist bdist_wheel
```

Comment:
@naturalett should we add a flavor which pushes to test pypi?

Reply:
1. Would it be better to push to test pypi for every push to master?
2. To push to pypi will be by a user demand.

WDYT?

@assapin

Reply:
test pypi on merge to master seems logical
"real" pypi is only for RC and official versions - triggered manually makes sense too for now